google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -112,7 +129,8 @@ public HttpStorageRpc(StorageOptions options) {
     private DefaultRpcBatch(Storage storage) {
       this.storage = storage;
       batches = new LinkedList<>();
-      batches.add(storage.batch());
+      // add OpenCensus HttpRequestInitializer
+      batches.add(storage.batch(batchRequestInitializer));
```

Comment:
`storage.batch()` can take an `HttpRequestInitializer` as parameter. If not given, it will use null.
Since the original code does not `initializer` for some reason, here I use another initializer which wrapper nothing to maintain the same logic.